pontoon/administration/static/css/admin_project.css

```diff
@@ -121,8 +122,8 @@ form a:link, form a:visited {
   text-transform: uppercase;
 }
 
-form a.add-inline,
-form a.add-repo {
+#admin-form form a.add-inline,
+#admin-form form a.add-repo {
```

Comment:
I think you meant:

```css
#admin-form a.add-inline,
#admin-form a.add-repo {
```

Otherwise "Add More" links on the project admin page grow in size.